### PR TITLE
Capture user operations on Playbook Generation UI

### PR DESCRIPTION
**Pull request by TamiTakamiya**
Related: https://issues.redhat.com/browse/AAP-24906

### CODE REVIEW

**Review by ssbarnea — CHANGES_REQUESTED** [review]
Do not merge this until @ganeshrn approves because it appears to work against reducing amount of analytics data. @ganeshrn remove my -1 if you think is ok.

See https://issues.redhat.com/browse/AAP-24906

**Review by ganeshrn — APPROVED** [review]
The analytics for lightspeed is send from the lightspeed service unlike the other features of extension that directly send the events to segement. The change in the PR looks good to me.

**Pull request merged**
